api/agent/drivers/docker/registry_test.go

```diff
@@ -1,7 +1,10 @@
 package docker
 
 import (
+	"strings"
 	"testing"
+
+	"github.com/fsouza/go-dockerclient"
```

Comment:
Is this a new dep?

Reply:
No, we use it currently in driver/docker.